cpp/oneapi/dal/backend/interop/sparse_block.hpp

```diff
@@ -0,0 +1,70 @@
+/*******************************************************************************
+* Copyright 2021 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#pragma once
+
+namespace oneapi::dal::backend::interop {
+
+template <typename T>
+struct sparse_block_owner {
+    sparse_block_owner(const daal::data_management::NumericTablePtr& nt) {
+        _csr_nt = daal::services::dynamicPointerCast<daal::data_management::CSRNumericTable,
+                                                     daal::data_management::NumericTable>(nt);
+        ONEDAL_ASSERT(_csr_nt);
+        _row_count = dal::detail::integral_cast<std::int64_t>(_csr_nt->getNumberOfRows());
+        _column_count = dal::detail::integral_cast<std::int64_t>(_csr_nt->getNumberOfColumns());
+        _csr_nt->getSparseBlock(0,
```

Comment:
Check the status and throw an exception?

Reply:
Ok, it was fixed